scripts/install/install_unix.sh

```diff
@@ -937,6 +937,10 @@ main() {
         base_url=$2
         shift 2
         ;;
+      -r | --uninstall)
```

Comment:
Should this be described in the usage() function as well?

Reply:
I'll do a follow up PR to branch `release/v1.65.0` bc it's not in there either and should be. After the next release this branch will get rebased and include the change.